packages/grafana-data/src/utils/location.ts

```diff
@@ -17,16 +17,10 @@ const stripBaseFromUrl = (url: string): string => {
   const isAbsoluteUrl = url.startsWith('http');
   let segmentToStrip = appSubUrl;
 
-  if (!url.startsWith('/')) {
+  if (!url.startsWith('/') || isAbsoluteUrl) {
     segmentToStrip = `${window.location.origin}${appSubUrl}`;
   }
 
-  if (isAbsoluteUrl) {
-    segmentToStrip = url.startsWith(`${window.location.origin}${appSubUrl}`)
-      ? `${window.location.origin}${appSubUrl}`
-      : `${window.location.origin}`;
-  }
```

Comment:
this condition seem wrong to me, when should we ever remove just the domain and not the domain & subpath? Removing just the domain will make links to other urls on the same domain impossible